golem_messages/message/tasks.py

```diff
@@ -267,11 +274,18 @@ def requestor_ethereum_address(self):
         )
 
     @property
-    def provider_ethereum_address(self):
-        return to_checksum_address(
-            sha3(decode_hex(self.provider_ethereum_public_key))[12:].hex(),
-        )
+    def provider_public_key(self):
+        return self.want_to_compute_task.provider_public_key
+
+    @property
+    def provider_ethereum_public_key(self):
+        return self.want_to_compute_task.provider_ethereum_public_key
+
+    @property
+    def eth_account(self):
```

Comment:
Why change property name from `provider_ethereum_address` to `eth_account`?

Reply:
:+1: done, you are right, it is actually unnecessary change.